mythril/interfaces/cli.py

```diff
@@ -331,9 +331,15 @@ def main():
 
         if args.code:
             # Load from bytecode
-            address, _ = mythril.load_from_bytecode(args.code, args.bin_runtime)
+            if args.code.startswith("0x"):
```

Comment:
What do you think about the following change?
```suggestion
           code = args.code[2:] if args.code.startswith("0x") else args.code
```